iuf

```diff
@@ -570,6 +620,7 @@ def main():
     validate_stages(config)
 
     atexit.register(log_state_files, config)
+    atexit.register(print_extra_summary, config)
```

Comment:
Not sure if it will cause a merge conflict, but I'll wait until this is merged, and rebase the SCICD-595 branch afterwards.  The atexit.register() calls were moved.